### PR TITLE
SCW-384: Create updated demo site for testing

### PR DESCRIPTION
**Pull request by lluiserena**
## Description

Created a next frontend inside the AA package to do the manual testing and start the integration of Playwright to test.

## Test plan

On `packages/client/wallets/aa/front` use the command `yarn dev` to open the frontend on `localhost:3001`

### CODE REVIEW

File: packages/client/wallets/aa/front/services/walletService.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+export const getOrCreateWallet = async () => {
+    const { CrossmintAASDK } = await import("@crossmint/client-sdk-aa");
+    const xm = CrossmintAASDK.init({
+        apiKey: process.env.NEXT_PUBLIC_API_KEY_STG || '',
+    });
+    const signer: ethers.Wallet = new ethers.Wallet("01f4f4a0efe13dbeaff3e2a030f0fa1754e2d90b1391a0120a6a11c1bf10ea6a");
+    const walletInitParams = {
+        signer
+    }
+    const userIdentifier = { email:"_test@test.com" };
```
Comment (MatiNavCrossmint) [inline packages/client/wallets/aa/front/services/walletService.ts:16]: This should be kind of dynamic / or received by input. If not, we will execute the creation flow the first time, and then always the retrieve one

**Review by MatiNavCrossmint — COMMENTED** [review]
Overall It looks good.

What we need to add, is having both buttons:
- "Execute getOrCreate with same email"
- "Execute getOrCreateWithNewEmail"

With that, the former should test the creation flow, the latter the retrieve, and also we will be sure that we are testing if the solution identifies wether to execute one or the other, successfully

File: packages/client/wallets/aa/front/services/walletService.ts (hunk 1 above)
Comment (lluiserena) [reply packages/client/wallets/aa/front/services/walletService.ts]: **Added two inputs** to add the `email` and the `private key` to create the wallet using `new ethers.Wallet(privateKey);`

File: packages/client/wallets/aa/front/pages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                <input
+                    className={styles.input}
+                    placeholder="Private key"
+                    value={privateKey}
+                    onChange={(e) => setPrivateKey(e.target.value)}
+                />
+                <button className={styles.button} onClick={handleCreateWallet}>
+                    Create wallet
+                </button>
+                <input className={styles.input} type="text" value={walletAddress} readOnly />
```
Comment (MatiNavCrossmint) [inline packages/client/wallets/aa/front/pages/index.tsx:88]: Is the `readinly` attribute ok here or will it make impossible to change the input's value?

**Review by MatiNavCrossmint — COMMENTED** [review]
Awesome, just 1 comment

File: packages/client/wallets/aa/front/pages/index.tsx (hunk 2 above)
Comment (lluiserena) [reply packages/client/wallets/aa/front/pages/index.tsx:88]: we won't want to change the walletAddress, we retrieve it from the getOrCreateWalletProcess and we show the user the address.

**Review by MatiNavCrossmint — APPROVED** [review]
LGTM!

**Pull request merged**